### PR TITLE
Chinese translation of  "Contributor" section.

### PR DESCRIPTION
**Pull request by niminghong**
contributor/zh

### CODE REVIEW

**Comment by rrrutledge** [comment]
Hi, @niminghong, I've tried to reach out in the InnerSource community for a reviewer for this content, but haven't run across anyone, yet.  Do you have a professional contact at another company that could review this work?

Thank you.

**Comment by tanzhongyibidu** [comment]
russ，I will take a look.

**Comment by lenucksi** [comment]
> russ，I will take a look.

Excellent, thanks a lot! :smiley:  :+1:

**Comment by rrrutledge** [comment]
Yes, thanks so much @tanzhongyibidu !!

File: contributor/zh/04-mechanics-of-contributing-zh.asciidoc [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# 贡献机制
+
+你准备好开始为其他团队项目/回购做出贡献了吗？你是否希望通过协作而不是通过管理升级来减少阻碍？本节给出了实用的建议，并重点介绍了在 为InnerSource 做出贡献时要记住的问题。它使你和项目团队有尽可能的愉快的体验，为更多的贡献和伟大的合作奠定基础。
```
Comment (adadai88) [inline contributor/zh/04-mechanics-of-contributing-zh.asciidoc:3]: 回购错误：should be repository， in Chinese is 仓库

File: contributor/zh/04-mechanics-of-contributing-zh.asciidoc [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+.  https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/04-mechanics-of-contributing-article.asciidoc#submitting-the-pull-request[_完善你的工作，并将其呈现给项目团队_]。
+
+当你的贡献和目标越来越多，你可能会重复地经历其中一些步骤。很有可能当你这样做的时候，一切都会变得越来越自然——也许你甚至会奇怪你什么时候做过那些步骤。
+
+### 准备工作
+
+#### 交付周期
+
+一个关键的区别是项目周期。每当你有第一次贡献的时候，你都会加入一个新的（项目）团队。因此，你需要了解他们的代码、所使用的技术以及他们喜欢的开发环境（比如测试框架、构建系统）。即使在这种类型的工具已标准化的情况下，每个团队也会开发出一些单独的特性。除了技术方面，你可能面临沟通上的分歧（比如代码评审）。即使当你过一段时间再回来，团队的工作方式和成员可能已经改变了。你得花点时间去了解一个你很久没见的朋友，因为你现在正要去拜访他。
```
Comment (adadai88) [inline contributor/zh/04-mechanics-of-contributing-zh.asciidoc:19]: 新的（项目）团队：  should be 新的（维护）团队
我们将host team and guest team翻译成专有名词：调用团队(guest team) ，称团队B为 维护团队(host team)

File: contributor/zh/04-mechanics-of-contributing-zh.asciidoc (hunk 1 above)
Comment (niminghong) [reply contributor/zh/04-mechanics-of-contributing-zh.asciidoc]: done

File: contributor/zh/04-mechanics-of-contributing-zh.asciidoc (hunk 2 above)
Comment (niminghong) [reply contributor/zh/04-mechanics-of-contributing-zh.asciidoc]: done

Comment (adadai88) [reply contributor/zh/04-mechanics-of-contributing-zh.asciidoc]: well done,  LGTM.

File: contributor/zh/04-mechanics-of-contributing-zh.asciidoc (hunk 1 above)
Comment (adadai88) [reply contributor/zh/04-mechanics-of-contributing-zh.asciidoc]: thanks Nini.LGTM

**Comment by adadai88** [comment]
I am ok with the mentioned translation docs ,thanks.

File: contributor/zh/01-introduction-zh.asciidoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# 介绍
+
+### InnerSource 的贡献者
```
Comment (tanzhongyi003) [inline contributor/zh/01-introduction-zh.asciidoc:3]: InnerSource 的贡献者（contributor）

File: contributor/zh/01-introduction-zh.asciidoc [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+### 与其他角色的关系
+
+作为 InnerSource 社区中的贡献者，你将与 InnerSource 的其他角色（例如 https://innersourcecommons.org/resources/learningpath/trusted-committer/index[_受信任提交者 Trusted Committer_] 或 https://innersourcecommons.org/resources/learningpath/product-owner/index[_产品所有者 Product Owner_] ）进行交互，也可能与其他贡献者进行交互。有时，这些角色可以由同一个人扮演，例如小型基础项目中的“受信任的提交者”和“产品所有者”。
+
+本节为你简要概述了其他两个角色，但我们想鼓励你去阅读 https://innersourcecommons.org/resources/learningpath/trusted-committer/index[_“受信任提交者”的介绍文章introductory article of the Trusted Committer role_] ，也推荐你在深入研究本节的“贡献者”角色之前阅读 https://innersourcecommons.org/resources/learningpath/trusted-committer/05/[_降低准入门槛 Lowering the barriers to entry_] 一文，你也可以观看视频（ https://innersourcecommons.org/resources/learningpath/trusted-committer/index[_介绍 introduction_]、 https://innersourcecommons.org/resources/learningpath/trusted-committer/05/[_降低准入门槛 Lowering the barriers to entry_]）而无需阅读文章。
+
+### 受信任提交者
+
+受信任提交者是社区的主人，他们对项目代码把关，审核你贡献的代码，并最快应用到生产环境。他们的职责是指引你如何为社区做贡献，直接帮助你，或者间接提供信息，例如社区审核规范、大规模修改建议，以及文档指引或者与你修改相关的代码片断。
+
```
Comment (tanzhongyi003) [inline contributor/zh/01-introduction-zh.asciidoc:20]: 等

File: contributor/zh/01-introduction-zh.asciidoc [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+他们还需要从技术和一般角度来关注产品质量、可持续性和项目发展，减少每个人为做出贡献所遇到的障碍，以及关注整个社区。关注社区包括保持社区健康，提升社区及其参与者的水平，并在组织中倡导社区的需求。
+
```
Comment (tanzhongyi003) [inline contributor/zh/01-introduction-zh.asciidoc:22]: 倡导感觉不太好，建议换个词。

**Comment by rrrutledge** [comment]
Thanks for these comments, @tanzhongyi003 !

**Comment by tanzhongyi003** [comment]
russ & lenuckis， please dont merge until I gave my LGTM review。  I have discussed with the contributor offline, they will provide a more updated version soon.

**Comment by rrrutledge** [comment]
Sounds good @tanzhongyi003

**Comment by tanzhongyi003** [comment]
I will take a look。

File: contributor/zh/01-introduction-zh.asciidoc [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # 介绍
 
-### InnerSource 的贡献者
+### InnerSource 的贡献者（contributor）
```
Comment (tanzhongyi003) [inline contributor/zh/01-introduction-zh.asciidoc:3]: contributor 最好首字母大写（Contributor）

File: contributor/zh/01-introduction-zh.asciidoc [hunk 7]
```diff
@@ -1,6 +1,6 @@
...
 
 你的工作计划是否曾经因为另一个团队没有做完你所依赖的功能而卡住？也许你不得不在项目中做一些额外的工作来解决这些缺少的功能。若能不再受这种问题困扰，那该有多美好呀？
```
Comment (tanzhongyi003) [inline contributor/zh/01-introduction-zh.asciidoc:5]: 解决缺少的功能， 听着还是不太顺，  要不改为“补上这些缺失的功能”？

File: contributor/zh/01-introduction-zh.asciidoc [hunk 8]
```diff
@@ -14,13 +14,13 @@
 
 本节为你简要概述了其他两个角色，但我们想鼓励你去阅读 https://innersourcecommons.org/resources/learningpath/trusted-committer/index[_“受信任提交者”的介绍文章introductory article of the Trusted Committer role_] ，也推荐你在深入研究本节的“贡献者”角色之前阅读 https://innersourcecommons.org/resources/learningpath/trusted-committer/05/[_降低准入门槛 Lowering the barriers to entry_] 一文，你也可以观看视频（ https://innersourcecommons.org/resources/learningpath/trusted-committer/index[_介绍 introduction_]、 https://innersourcecommons.org/resources/learningpath/trusted-committer/05/[_降低准入门槛 Lowering the barriers to entry_]）而无需阅读文章。
 
-### 受信任提交者
+### 受信任提交者（Trusted Committer）
```
Comment (tanzhongyi003) [inline contributor/zh/01-introduction-zh.asciidoc:17]: 按照之前从Russ哪里得到的信息，Trusted Committer， 不翻译为中文，直接保留就好。

Comment (niminghong) [reply contributor/zh/01-introduction-zh.asciidoc]: done，其他地方的Trusted Committer也改好了。

**Comment by tanzhongyi003** [comment]
OK，I will take a look again.

**Comment by tanzhongyi003** [comment]
This is the first contribution of @niminghong for github, cheer.

**Comment by niminghong** [comment]
> > LGTM，thanks niminghong.
> 
> Excellent, will merge now. Thanks a lot to everyone for the great translation work and the great reviews @niminghong @tanzhongyi003 @adadai88 🎉

My pleasure:)

**Comment by niminghong** [comment]
> This is the first contribution of @niminghong for github, cheer.

Thx,cheer:)

**Pull request merged**